packages/legend-studio/src/components/editor/edit-panel/uml-editor/ClassEditor.tsx

```diff
@@ -1482,6 +1481,7 @@ export const ClassFormEditor = observer(
             flex={0}
             direction={-1}
             size={editorState.selectedProperty ? 250 : 0}
+            minSize={editorState.selectedProperty ? 250 : 0}
```

Comment:
We can't do this, `minSize` will make it not possible for people to collapse by `drag-and-drop` if I understood correctly. I think the approach is to look at the API of `react-reflex` a bit here.

Also, this code pattern is being used in several places, perhaps, it's best to (whatever change you do here) apply the same change to those places in the end